setup.py

```diff
@@ -1,38 +0,0 @@
-from setuptools import setup, find_packages
-from os import path
-from io import open
-
-import versioneer
-
-here = path.abspath(path.dirname(__file__))
-
-# Get the long description from the README file
-with open(path.join(here, 'README.md'), encoding='utf-8') as f:
-    long_description = f.read()
-
-setup(name='pynvml',
-      version=versioneer.get_version(),
-      cmdclass=versioneer.get_cmdclass(),
-      python_requires='>=3.6',
-      description='Python utilities for the NVIDIA Management Library',
-      long_description=long_description,
-      long_description_content_type='text/markdown',
-      packages=find_packages(exclude=['notebooks', 'docs', 'tests']),
-      package_data={'pynvml': ['README.md','help_query_gpu.txt']},
```

Comment:
Seems `help_query_gpu.txt` isn't packaged in the current setup. `README.md` is already

Think we just need a one line `MANIFEST.in` like so

```
include help_query_gpu.txt
```